x/liquidity/spec/05_begin_block.md

```diff
@@ -1,14 +1,12 @@
-<!--
-order: 5
--->
+<!-- order: 5 -->
 
 # Begin-Block
```

Comment:
okay, here's a new document... but what is it? what is a begin-block and why do we document it? can we describe it?

Reply:
today I learned about the spec spec!